containers.go

```diff
@@ -297,7 +311,7 @@ func (r *containerStore) Create(id string, names []string, image, layer, metadat
 			BigDataSizes:   make(map[string]int64),
 			BigDataDigests: make(map[string]digest.Digest),
 			Created:        time.Now().UTC(),
-			Flags:          make(map[string]interface{}),
+			Flags:          options.Flags,
```

Comment:
Use `copyStringInterfaceMap(options.Flags)` here.  It attempts to copy the map to avoid possible shenanigans if the passed-in map changes later.

Reply:
Fixed